tf_agents/agents/behavioral_cloning/behavioral_cloning_agent.py

```diff
@@ -256,7 +256,8 @@ def _continuous_loss_fn(self, experience, training: bool = False):
     else:
       bc_action = bc_output
 
-    losses = tf.nest.map_structure(tf.losses.mse, experience.action, bc_action)
+    losses = tf.nest.map_structure(lambda loss, a, b: loss(a,b), 
```

Comment:
Add test with multiple actions.

Reply:
@sguada Do you want me to add more functionality to this commit? I can remove the check for single, scalar discrete actions.
Right now, Multiple Actions are not allowed with the built-in loss functions, and I think adding them can probably be a separate commit.